packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardActivity.styles.ts

```diff
@@ -0,0 +1,109 @@
+import { getGlobalClassNames, FontSizes, FontWeights } from '../../Styling';
+import { IDocumentCardActivityStyleProps, IDocumentCardActivityStyles } from './DocumentCardActivity.types';
+
+const VERTICAL_PADDING = 8;
```

Comment:
Just for future reference, these consts would be "tokens" to allow them to be configurable.